dbms/src/Storages/DeltaMerge/DeltaMergeStore_InternalBg.cpp

```diff
@@ -700,11 +700,7 @@ SegmentPtr DeltaMergeStore::gcTrySegmentMerge(const DMContextPtr & dm_context, c
     auto segment_bytes = segment->getEstimatedBytes();
     if (segment_rows >= dm_context->segment_limit_rows || segment_bytes >= dm_context->segment_limit_bytes)
     {
-        LOG_TRACE(
-            log,
-            "GC - Merge skipped because current segment is not small, segment={} table={}",
-            segment->simpleInfo(),
-            table_name);
+        LOG_TRACE(log, "GC - Merge skipped because current segment is not small, segment={}", segment->simpleInfo());
```

Comment:
Note: The logger of `DeltaMergeStore` contains the keyspace and table_id as a suffix. So we don't need to log down the table_name in these methods.
https://github.com/pingcap/tiflash/blob/ce3d9e124e7949d053603c0e6f7be5b30d64a14d/dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp#L231